aip/auth/4110.md

```diff
@@ -138,13 +146,15 @@ digraph d_front_back {
     1. If certificate is presented, embed the certificate into the JWT.
     1. Use the regular [self-signed JWT flow][4] for an access token. _[END]_
   1. Use user identity flow to exchange for an access token. _[END]_
+  1. Use external account (via `sts.googleapis.com`) flow to exchange for an access token. _[END]_
```

Comment:
Can we link the external account flow here to AIP 4117?

Reply:
Done